### PR TITLE
Feat: Added Preloader in edit post successfully Issue 363

### LINKED ISSUE #363: [Bug]: Instead of Preloader hardcoded loading is showing when going in edit post
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Instead of Preloader hardcoded loading is showing when going in edit post.
So, I will replace the loading... text with the style share preloader.

### Expected behavior

Loading behavior

### Add ScreenShots

image

### On which device are you experiencing this bug?

_No response_

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I have starred the repository

### PR DESCRIPTION
**Pull request by MeetDOD**
# Pull Request Resolves [#363 ]

### Description

1. Added the preloader as Loader (from components) in the edit post page.

### Related Issues
Fixes #363 

### Changes Made
New features: Added Preloader in edit post.

I certify that I have carried out the relevant code of conduct and provided the requisite screenshot for validation by submitting this pull request.

Thank You for this contribution.

### CODE REVIEW

**Review by VaibhavArora314 — COMMENTED** [review]
looks good to me, just don't push changes of your other pull request

**Comment by MeetDOD** [comment]
@VaibhavArora314 I was trying to avoid it at all the cost but it happens sometime by mistakenly :(

**Comment by MeetDOD** [comment]
@akbatra567 @Ultimateutkarsh11 Please review this pr, if any issue let me know 😄

**Review by Ultimateutkarsh11 — APPROVED** [review]

**Comment by MeetDOD** [comment]
Done

**Comment by MeetDOD** [comment]
@akbatra567 please review this PR 🙏

File: frontend/src/pages/Signin.tsx [hunk 1]
```diff
@@ -23,6 +23,7 @@ const Signin = () => {
   const setTokenState = useSetRecoilState(tokenState);
   const navigate = useNavigate();
 
+  document.title='Style Share | Login page 👋'
```
Comment (akbatra567) [inline frontend/src/pages/Signin.tsx:26]: remove this duplicated line.

**Review by akbatra567 — CHANGES_REQUESTED** [review]

File: frontend/src/pages/Signin.tsx (hunk 1 above)
Comment (MeetDOD) [reply frontend/src/pages/Signin.tsx]: Done @akbatra567

**Review by VaibhavArora314 — CHANGES_REQUESTED** [review]
captcha is twice on sign in page

File: frontend/src/pages/Signin.tsx
[the remark this answers is not in the document]
Comment (MeetDOD) [reply frontend/src/pages/Signin.tsx:1]: image

@VaibhavArora314  Where it is ? Everything is fine !

[the remark this answers is not in the document]
Comment (VaibhavArora314) [reply frontend/src/pages/Signin.tsx:1]: image

File: frontend/src/pages/Signin.tsx [hunk 2]
```diff
@@ -112,8 +112,8 @@ const Signin = () => {
             <p className="text-sm font-semibold mb-2 text-red-600">
               {error.password}
             </p>
-            <CaptchaUser onChange={(isValid) => setIsCaptchaValid(isValid)} /> {/* Add Captcha component */}
           </div>
+          <CaptchaUser onChange={(isValid) => setIsCaptchaValid(isValid)} />
           <div className="flex justify-center">
```
Comment (akbatra567) [inline frontend/src/pages/Signin.tsx:117]: remove this duplicate code.

**Review by akbatra567 — CHANGES_REQUESTED** [review]

File: frontend/src/pages/Signin.tsx
[the remark this answers is not in the document]
Comment (MeetDOD) [reply frontend/src/pages/Signin.tsx:1]: @VaibhavArora314 Done the changes check now !

**Review by VaibhavArora314 — APPROVED** [review]

File: frontend/src/pages/Signin.tsx (hunk 2 above)
Comment (MeetDOD) [reply frontend/src/pages/Signin.tsx:117]: @akbatra567  already did this

**Comment by MeetDOD** [comment]
@akbatra567 review this pr because i want to work on other issues also, if you are available 👍

**Review by akbatra567 — APPROVED** [review]

**Pull request merged**